### ISSUE TITLE
Step 4: Recipient Key

### ISSUE THREAD

**Issue by sethhrbek** [sarcophagus-org/sarcophagus-v2-app#24]
**Overview**
Implement the "Recipient Key" step -- this will differ slightly from the designs.

I'm not sure the best way to implement the UX for this step. Taking a stab at it below, but if you think there is a better UX flow, feel free to modify.

**Tasks**
1. Setup 2 radio buttons with labels, side by side. One says "Recipient Address", and one says "Recipient Public Key". Neither should be selected, and text above these should say "Please select one"

2. If the user selects "Recipient Address" radio button
a. Show a the recipient address input.
b. When the user inputs an eth address, validate it is a valid address
c. if it is valid, see if a public key can be derived from the address
d. if so, show the user a "success" text with a checkmark (or something to indicate we derived the public key).
e. at this point, the public key "requirement" section in the navigation would also get checked (may need to add this functionality later once requirements is done)
f. If we *cant* derive the public key, tell the user we were unable to derive the public key, and tell them they will need to either get the public key from the recipient, or sign into the sarcophagus app as the recipient and follow a link. This link is the same link as in v1, which will allow the user to generate the recipient's public key. We will need to copy over this public key generation functionality, but can do that in another task. For now, just have a link that opens a new browser window to a stubbed out route.

3. If the user selects "Recipient Public Key" radio button
a. Show the user the recipient public key input
b. When the user pastes in a recipient public key, validate the public key (can use same validation as was used in v1)

4. If the user switches radio buttons (i.e. is on "recipient's address", but switches to "recipient's public key"), it should clear out both inputs in state (address / pub key).
